packages/orchestration/src/examples/send-anywhere.contract.js

```diff
@@ -5,9 +5,10 @@ import { prepareChainHubAdmin } from '../exos/chain-hub-admin.js';
 import { AnyNatAmountShape } from '../typeGuards.js';
 import { withOrchestration } from '../utils/start-helper.js';
 import { registerChainsAndAssets } from '../utils/chain-hub-helper.js';
-import * as flows from './send-anywhere.flows.js';
 import * as sharedFlows from './shared.flows.js';
 
+import * as cctpFlows from './send-cctp.flows.js';
```

Comment:
I think `send-cctp.flows.js` was a SPIKE approach. I think the goal is to integrate it back into `./send-anywhere.flows.js`.